apps/judicial-system/web/src/routes/Shared/Cases/Cases.tsx

```diff
@@ -41,57 +37,15 @@ import CasesAwaitingConfirmationTable from '../../Prosecutor/components/CasesAwa
 import CasesAwaitingReview from '../../PublicProsecutor/Tables/CasesAwaitingReview'
 import ActiveCases from './ActiveCases'
 import { useCasesQuery } from './cases.generated'
+import { CreateCaseButton } from './CreateCaseButton'
 import { FilterOption, useFilter } from './useFilter'
 import { cases as m } from './Cases.strings'
 import * as styles from './Cases.css'
 
-interface CreateCaseButtonProps {
+export interface CreateCaseButtonProps {
```

Comment:
Is this only used in the `CreateCaseButton` component? Consider moving this there and rename it to `Props`